totalspineseg/utils/cpdir.py

```diff
@@ -141,18 +150,42 @@ def cpdir_mp(
 def _cpdir(
         src_path,
         dst_path,
-        override=False,
+        num_tries=1,
+        overwrite=False,
     ):
     '''
     Copy file from src to dst.
     '''
     # If the output already exists and we are not overriding it, return
-    if not override and dst_path.exists():
+    if not overwrite and dst_path.exists():
         return
 
-    # Make sure dst directory exists and copy
+    # Make sure dst directory exists
     dst_path.parent.mkdir(parents=True, exist_ok=True)
-    shutil.copy2(src_path, dst_path)
+
+    # Try copying up to num_tries times
+    for attempt in range(num_tries):
```

Comment:
If it doesn't work the first time, is possible that it will work after ?

Reply:
Yes
In network drive like in compute canada